### DISCUSSION TITLE
Water Walls Outer Bounds

### DISCUSSION THREAD

**Discussion by particl3s** [OpenRCT2/OpenRCT2#20401] [Feature requests] [answered]
Currently outer walls occupied by water reflect the landmass, this feels unnatural. If at all possible would get a look & feel feature in making water transparent. 

[![2023-06-15-krita-water-walls-proposal-01.png](https://i.postimg.cc/xfxccpVm/2023-06-15-krita-water-walls-proposal-01.png)](https://postimg.cc/XBVSG9dt)
[![2023-06-15-krita-water-walls-proposal-02.png](https://i.postimg.cc/92J480wf/2023-06-15-krita-water-walls-proposal-02.png)](https://postimg.cc/qg0d5HSQ)

I like this idea because it makes Aquarium like environments & could shine with multiple enable/disable area option.

**Comment by ZehMatt** [comment]
It's an interesting idea and I like it, the problem is that there are currently no sprites that would allow doing this.

**Reply by ocalhoun6** [reply, marked as answer]
Seems like it shouldn't be *that* hard to make sprites for it, though ... right?

Basically just need a vertical wall, translucent, with the 'dark water' color. And with the way palates work, it will automatically be recolored when people are using a different water color.

The only slight difficulty is that you might have to make three different sprites for it: flat, up slope, and down slope, with each being exactly one full unit tall. Then you just need the code to stack those three sprites up the right way along any applicable edges. And, of course, redraw it differently when the view is rotated.

But still, not much actual *art* needed for it. It doesn't need to have waves or anything. And it does look pretty cool in that image above -- I'd love to see it become real!


---------

Edit: Oh, and since 'transparent water' is already an option, we might want a separate set of sprites to use for the opaque water. Not absolutely necessary (I think it would still be fine if you could see through the vertical water edge while the top of the water was opaque), but adding an extra set of sprites for opaque water (same as the others, just without transparency) would really round out the feature and provide nice attention to detail.

**Reply by ZehMatt** [reply]
Something to keep in mind is that when using the palette for water it will be animated, its probably better to base it on the glass ones or it will look like the surface of it.
